Cognite.Jetfire.Test/Deploy/Manifest/ManifestParserTest.cs

```diff
@@ -80,6 +80,8 @@ public void NotSharedByDefault()
         [InlineData(DestinationType.StringDatapoints, "stringDatapoints")]
         [InlineData(DestinationType.Files, "files")]
         [InlineData(DestinationType.Sequences, "sequences")]
+        [InlineData(DestinationType.Assets, "labels")]
+        [InlineData(DestinationType.Assets, "relationships")]
```

Comment:
Remember to update the expected value here 🙂

Reply:
Ah sure :D